lib/internal/process.js

```diff
@@ -70,10 +70,48 @@ function setupConfig(_source) {
 
 function setupKillAndExit() {
 
-  process.exit = function(code) {
+  const kExitTimer = Symbol('kExitTimer');
```

Comment:
tiniest nit: it's probably historic here also, but can we avoid k-prefied variable names, since `const` is common nowadays? See, [here](http://stackoverflow.com/a/5017113/3580261).

I like the idea of this though.